server/routes/api.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const app = express()
+const router = express.Router()
+const Image = require('../models/Image')
+
+router.get('/images', async (req, res) => { 
+    const {id} = req.query
+    const images = id ? await Image.findById(id) : await Image.find({})
+    res.send(images)
+})
+
+router.post('/image', async (req, res) => {
+    let result
+    try{
+        const newImage = new Image(req.body)
+        await newImage.save()
+        result = "The image was saved"
```

Comment:
do not send strings send status codes and maybe the updated image document from the db